test/1_stdlib/VarArgs.swift

```diff
@@ -48,10 +46,10 @@ func test_varArgs3() {
   args.append(COpaquePointer(bitPattern: 0x1234_5671))
   args.append(UnsafePointer<Int>(bitPattern: 0x1234_5672))
   args.append(UnsafeMutablePointer<Float>(bitPattern: 0x1234_5673))
-  args.append(AutoreleasingUnsafeMutablePointer<AnyObject>(
-        UnsafeMutablePointer<AnyObject>(bitPattern: 0x1234_5674)))
+  //args.append(AutoreleasingUnsafeMutablePointer<AnyObject>(
+  //      UnsafeMutablePointer<AnyObject>(bitPattern: 0x1234_5674)))
```

Comment:
Please don't remove the test.  Instead, conditionalize it on `#if _runtime(_ObjC)`.